pkgs/applications/misc/openrgb/default.nix

```diff
@@ -2,18 +2,26 @@
 
 mkDerivation rec {
   pname = "openrgb";
-  version = "0.3";
+  version = "0.4";
 
   src = fetchFromGitHub {
     owner = "CalcProgrammer1";
     repo = "OpenRGB";
     rev = "release_${version}";
-    sha256 = "1931aisdahjr99d4qqk824ib4x19mvhqgqmkm3j6fc5zd2hnw87m";
+    sha256 = "sha256-tHrRG2Zx7NYqn+WPiRpAlWA/QmxuAYidENanTkC1XVw";
   };
 
   nativeBuildInputs = [ qmake pkg-config ];
   buildInputs = [ libusb1 hidapi ];
 
+  # For reproducible builds
+  # This change has been accepted upstream so this can be removed next release (0.5)
+  # https://gitlab.com/CalcProgrammer1/OpenRGB/-/commit/f1b7b8ba900db58a1119d8d3e21c1c79de5666aa
+  postPatch = ''
+    substituteInPlace OpenRGB.pro \
+    --replace 'date -R' 'date -R -d "@''${SOURCE_DATE_EPOCH:-$(date +%s)}"'
+  '';
```

Comment:
```suggestion
  patches = [
    # Make build SOURCE_DATE_EPOCH aware, merged in master
    (fetchpatch {
      url = "https://gitlab.com/CalcProgrammer1/OpenRGB/-/commit/f1b7b8ba900db58a1119d8d3e21c1c79de5666aa.patch";
      sha256 = "17m1hn1kjxfcmd4p3zjhmr5ar9ng0zfbllq78qxrfcq1a0xrkybx";
    })
  ];
```

Reply:
and please add `fetchpatch` to the inputs

Reply:
Ah, yes, should've thought of this myself, sorry.